packages/api/test/unit/beacon/testData/beacon.ts

```diff
@@ -9,8 +9,8 @@ import {
 } from "../../../../src/beacon/routes/beacon/index.js";
 import {GenericServerTestCases} from "../../../utils/genericServerTest.js";
 
-const root = Buffer.alloc(32, 1);
-const randao = Buffer.alloc(32, 1);
+const root = new Uint8Array(32).fill(1);
+const randao = new Uint8Array(32).fill(1);
```

Comment:
**One observation for reviewers**, it seems that `chai` was matching the `Buffer` and `Uint8Array` equality. Which is logically wrong, if we are expecting a `Buffer` instance of a value then assertion should pass otherwise fail. 

For this reason have to update a lot of fixture data in this PR, converting `Buffer` to actual returning `Uint8Array` instances.